src/combinatorics/young_diagram.lean

```diff
@@ -63,6 +66,8 @@ instance : set_like young_diagram (ℕ × ℕ) :=
 @[simp] lemma mem_cells {μ : young_diagram} (c : ℕ × ℕ) :
   c ∈ μ.cells ↔ c ∈ μ := iff.rfl
 
+instance decidable_mem (μ : young_diagram) : decidable_pred (∈ μ) := λ _, μ.cells.decidable_mem _
```

Comment:
This is another way you can write it, where you don't have to actually name the decidable instance you're using:
```suggestion
instance decidable_mem (μ : young_diagram) : decidable_pred (∈ μ) :=
show decidable_pred (∈ μ.cells), by apply_instance
```